interfaces/cython/cantera/examples/onedim/flame_fixed_T.py

```diff
@@ -25,7 +27,7 @@
 # transport properties. It is created with two transport managers, to enable
 # switching from mixture-averaged to multicomponent transport on the last
 # solution.
-gas = ct.Solution('gri30.xml', 'gri30_mix')
+gas = ct.Solution("gri30.yaml", "gas", "Mix")
```

Comment:
> The phase name in gri30.yaml is still gri30 (which I'd support leaving as-is), so gas won't work here

Yup, brain fart on my part 🤦‍♂️

> Also, if the default phase uses mixture-averaged transport, can't this just be gas = ct.Solution("gri30.yaml")?

My thought was to be explicit, both to show that the option was available (and I agree that the keyword should be included) as well as to avoid any issues in the future if the default transport model should change again.